lib/novelsetting.rb

```diff
@@ -390,7 +390,7 @@ def self.get_original_settings
       type: :string,
       value: "(%-m/%-d)",
       help: <<-EOS
-enable_add_date_to_title で付与する日付のフォーマット。書式は http://bit.ly/1m5e3w7 を参照。
+enable_add_date_to_title で付与する日付のフォーマット。書式は http://bit.ly/date_format を参照。
```

Comment:
__[RuboCop]__

Layout/IndentHeredoc: Use 2 spaces for indentation in a heredoc by using `<<~` instead of `<<-`.


<!-- Comment by [Sider](https://sider.review) -->